source/guide_privatebin.rst

```diff
@@ -0,0 +1,204 @@
+.. highlight:: console
+
+.. author:: Nepomacs <https://github.com/Nepomacs/>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: privacy
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: About
+
+  .. image:: _static/images/privatebin.png
+      :align: center
+
+##########
+PrivateBin
+##########
+
+.. tag_list::
+
+PrivateBin is a minimalist, open source online pastebin where the server has zero knowledge of pasted data.
+
+Data is encrypted and decrypted in the browser using 256bit AES in Galois Counter mode.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+PrivateBin consists of PHP and JS code which was originally written by Sébastien Sauvage in 2012 and falls unter the Zlib/libpng license.
+All relevant legal information can be found in the Github_ repository of the project.
+
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.4:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'PHP' version: '7.4'
+ [isabell@stardust ~]$
+
+
+
+The domain you want to use must be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Download the source
+-------------------
+
+Check Github_ for the `latest release`_ and copy the download link to the .tar.gz file.
+Then ``cd`` to your :manual:`DocumentRoot <web-documentroot>` and use ``wget`` to download it. Replace the URL with the one you just copied.
+In the example the version is 1.2.4 which does not exist.
```

Comment:
We had fake versions for the longest time, but it just ended up causing more harm than joy. We've updated the style guide since to say

> If possible use a download URL that points to the latest version (e.g. latest.zip on some platforms). If such an URL is not available, use the newest version instead.

Please update the guide to point to version 1.3.4.